src/sdgio-edit.owl

```diff
@@ -1608,6 +1638,235 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020331> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020331> "2019-06-04T17:56:54Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020331> "planned process to increase productivity or production capacity of agents"@en)
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020331> <http://purl.unep.org/sdg/SDGIO_00020335>)
+
+# Class: <http://purl.unep.org/sdg/SDGIO_00020332> (process to increase research capacity)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.unep.org/sdg/SDGIO_00020332> "A planned process during which an agent causes or contributes to increase the research capacity of an agent or group of agents.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020332> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020332> "2019-06-04T17:57:19Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020332> "process to increase research capacity"@en)
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020332> <http://purl.unep.org/sdg/SDGIO_00020331>)
+
+# Class: <http://purl.unep.org/sdg/SDGIO_00020334> (planned process to manage natural resources)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.unep.org/sdg/SDGIO_00020334> "A planned process during which an agent causes or contributes to the utilization or protection of either land, water, soil, plants, animals, or entire ecosystems.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020334> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020334> "2019-06-04T18:46:21Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020334> "planned process to manage natural resources"@en)
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020334> <http://purl.unep.org/sdg/SDGIO_00020335>)
+
+# Class: <http://purl.unep.org/sdg/SDGIO_00020335> (planned process to manage resources)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.unep.org/sdg/SDGIO_00020335> "A planned process during which an agent causes or contributes to the utlization or protection of resources.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000116> <http://purl.unep.org/sdg/SDGIO_00020335> "The SDGIO contains an entry \"resource\". It is appropriate to link that entry to this entry. However, the entry \"resource\" does not have a PURL.")
```

Comment:
@Mttbnchtt not sure what you meant here. http://purl.unep.org/sdg/SDGIO_00000013 is the PURL for 'resource' in SDGIO

Reply:
Thanks for that. I changed the editor note for the process. I did not see, and still do not see, that PURL in Protege. I am probably looking in the wrong place. Where did you find the PURL?

Reply:
You click on a class and press command and U (on mac), then click on the box "see full URI" something like that

Reply:
I linked "planned process to manage resouces" to "resource" via the axiom "has participant some resource".